python/src/CombinationsDistribution_doc.i.in

```diff
@@ -0,0 +1,89 @@
+%feature("docstring") OT::CombinationsDistribution
+"Combinations distribution.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+k : int, :math:`k > 0`
+n : int, :math:`n > 0`
+
+See also
+--------
+:class:`~openturns.Combinations`
+
+Notes
+-----
+This class is the discrete uniform distribution on the set of strictly increasing
+functions :math:`(i_0, \hdots, i_{k_1})`
+from :math:`\{0, \hdots, k-1\}` into :math:`\{0, \hdots, n-1\}`.
+Its probability distribution function is defined as:
+
+.. math::
+
+    \Prob{\vect{X} = (i_0, \hdots, i_{k-1})} = \frac{1}{\binom{n}{k}}
+
+Its first moments are:
+
+.. math::
+    :nowrap:
+
+    \begin{eqnarray*}
+        \Expect{\vect{X}} & = & \frac{n - 1}{2}\\
+        \Cov{\vect{X}} & = & \left\{
```

Comment:
So which of the two covariance expressions should we choose: the first one or the second one? Does it depend on the fact that n is even or odd?

Reply:
that's a bad copy-paste, to remove